File: Examples/CSharp/AsposePdfFacades/Text/AddText.cs
//////////////////////////////////////////////////////////////////////////
// Copyright 2001-2014 Aspose Pty Ltd. All Rights Reserved.
//
// This file is part of Aspose.Pdf. The source code in this file
// is only intended as a supplement to the documentation, and is provided
// "as is", without warranty of any kind, either expressed or implied.
//////////////////////////////////////////////////////////////////////////
using System.IO;

using Aspose.Pdf;
using Aspose.Pdf.Facades;

namespace CSharp.AsposePDFFacades.Text
{
    public class AddText
    {
        public static void Run()
        {
            // The path to the documents directory.
            string dataDir = RunExamples.GetDataDir_AsposePdfFacades_Text();

            //open document
            PdfFileMend mender = new PdfFileMend();

            //create PdfFileMend object to add text
            mender.BindPdf(dataDir + "AddText.pdf");

            //create formatted text
            FormattedText text = new FormattedText("Aspose - Your File Format Experts!", System.Drawing.Color.AliceBlue, System.Drawing.Color.Gray, Aspose.Pdf.Facades.FontStyle.Courier, EncodingType.Winansi, true, 14);

            //set whether to use Word Wrap or not and using which mode
            mender.IsWordWrap = true;
            mender.WrapMode = WordWrapMode.Default;

            //add text in the PDF file
            mender.AddText(text, 1, 100, 200, 200, 400);

            //save changes
            mender.Save(dataDir + "AddText_out.pdf");

            //close PdfFileMend object
            mender.Close();
            
        }
    }
}

File: Examples/VisualBasic/AsposePdfFacades/Text/AddText.vb
Imports System.IO
Imports System
Imports Aspose.Pdf
Imports Aspose.Pdf.Facades
Namespace AsposePDFFacades.Text
    Public Class AddText
        Public Shared Sub Run()
            ' ExStart:AddText
            ' The path to the documents directory.
            Dim dataDir As String = RunExamples.GetDataDir_AsposePdfFacades_Text()

            ' Open document
            Dim mender As New PdfFileMend()

            ' Create PdfFileMend object to add text
            mender.BindPdf(dataDir & Convert.ToString("AddText.pdf"))

            ' Create formatted text
            Dim text As New FormattedText("Aspose - Your File Format Experts!", System.Drawing.Color.AliceBlue, System.Drawing.Color.Gray, Aspose.Pdf.Facades.FontStyle.Courier, EncodingType.Winansi, True, _
                14)

            ' Set whether to use Word Wrap or not and using which mode
            mender.IsWordWrap = True
            mender.WrapMode = WordWrapMode.[Default]

            ' Add text in the PDF file
            mender.AddText(text, 1, 100, 200, 200, 400)

            ' Save changes
            mender.Save(dataDir & Convert.ToString("AddText_out_.pdf"))

            ' Close PdfFileMend object
            mender.Close()
            ' ExEnd:AddText
        End Sub
    End Class
End Namespace